community/lucene-index/src/main/java/org/neo4j/kernel/api/impl/labelscan/LuceneLabelScanStore.java

```diff
@@ -137,6 +137,12 @@ public void start() throws IOException
     }
 
     @Override
+    public boolean isEmpty() throws IOException
+    {
+        return allNodeLabelRanges().maxCount() == 0;
```

Comment:
Hmm wait, the `allNodeLabelRanges()` should be in a try-with-resource, will fix that